src/test/store/profile-view.test.js

```diff
@@ -216,14 +217,20 @@ describe('getJankMarkersForHeader', function() {
   });
 
   it('will create a jank instance with eventDelay values', function() {
-    const breakingPoint = 70;
-    const eventDelay = [0, 20, 40, 60, breakingPoint, 0, 20, 40];
+    const breakingPoint = 50;
+    // We need to fill the samples before, because the new algorithm is different
```

Comment:
The comment `The new algorithm is different` isn't very useful to a reader. Please be a bit more precise about what this means.